docs/modules/plugins/partials/plugin-web-app-steps.adoc

```diff
@@ -84,6 +84,41 @@ Given I am on main application page
 When I go to relative URL `/products/new`
 ----
 
+=== Web page steps
+
+==== Refresh page
+
+Refreshes the page. Browser reloads current page which is the same action as one caused by pressing F5 on the keyboard.
+
+[source,gherkin]
+----
+When I refresh page
+----
+
+==== Verify page title
+
+Checks that the page's *title* matches to *text* according to the provided string validation rule
+
+[source,gherkin]
+----
+Then page title $comparisonRule `$text`
+----
+
+- List of comparison rules: `is equal to`, `contains`, `does not contain`
+- `text` The text of the title to compare
```

Comment:
? please checks examples of the step input arguments in this doc

Reply:
Changed